casper/src/test/scala/coop/rchain/casper/ValidateTest.scala

```diff
@@ -791,6 +791,52 @@ class ValidateTest
       } yield result
   }
 
+  "Block validDeploySignatures" should "succeed if all deploys are signed correctly" in withStorage {
+    implicit blockStorey =>
+      implicit blockDagStorage =>
+        // given
+        val (sk, pk) = Secp256k1.newKeyPair
+        val block    = MultiParentCasperTestUtil.createGenesis(Map(pk -> 1))
+        for {
+          dag     <- blockDagStorage.getRepresentation
+          deploy1 <- ConstructDeploy.basicProcessedDeploy[Task](0, sk)
+          deploy2 <- ConstructDeploy.basicProcessedDeploy[Task](0, sk)
```

Comment:
If you keep the deploy contents all as 0, then it will be as if you only deployed 1.

Reply:
see i never knew what `0` stands for, good to know, thx!